firebase-firestore/src/main/java/com/google/firebase/firestore/AggregateQuerySnapshot.java

```diff
@@ -108,8 +103,6 @@ public long get(@Nonnull AggregateField.CountAggregateField countAggregateField)
    * @param averageAggregateField The average aggregation for which the value is requested.
    * @return The result of the given average aggregation.
    */
-  // TODO(sumavg): Remove the `hide` and scope annotations.
-  /** @hide */
   @RestrictTo(RestrictTo.Scope.LIBRARY)
```

Comment:
I think this RestrictTo should be removed. This member should be in the public API.